### PR TITLE
Timestamp vs timestamptz explanation

### LINKED ISSUE #3482: [Content Bug] Create Hypertable on timestamp (non-tz column). Supported?
_Use this template for reporting bugs in the docs._

# Describe the bug

Is partitioning by timestamp supported?

## What do the docs say now?

timestamp is supported / not supported. Depends on where you are looking at?

_not supported_
https://docs.timescale.com/use-timescale/latest/hypertables/create/#create-a-hypertable

`This can be a timestamptz, date, or integer. Ensure that you set the datatype for the time column as timestamptz and not timestamp.`

_supported_
https://docs.timescale.com/api/latest/hypertable/dimension_info/#notes

## What should the docs say?

consistent guidenance on what column types are supported.

# Page affected
https://docs.timescale.com/use-timescale/latest/hypertables/create/#create-a-hypertable
https://docs.timescale.com/api/latest/hypertable/dimension_info/#notes

# Subject matter expert (SME)

[If known, who is a good person to ask about this topic]

# Screenshots

[Attach images of screenshots showing the bug]

# Any further info

[Anything else you want to add, or further links]

## Contributing to documentation

We welcome documentation contributions!

*   For information about how to propose a change, see the [contributing guide](https://github.com/timescale/docs/blob/latest/CONTRIBUTING.md) in our GitHub repository.
*   For information on style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs)

### PR DESCRIPTION
**Pull request by atovpeko**

### CODE REVIEW

File: use-timescale/hypertables/create.md [hunk 1]
```diff
@@ -24,10 +24,8 @@ available](/api/:currentVersion:/hypertable/create_hypertable_old/).
 To create a hypertable, you need to create a standard PostgreSQL table, and then
 convert it into a hypertable.
 
-Hypertables are intended for time-series data, so your table needs a column that
-holds time values. This can be a timestamptz, date, or integer. Ensure that you
-set the datatype for the `time` column as `timestamptz` and not `timestamp`. For
-more information, see [PostgreSQL timestamp][postgresql-timestamp].
+Hypertables are designed for real-time analytics and partitioned by columns that hold time values. These can be of the `timestamptz`, `date`, or `integer` types. While `timestamp` is also supported, a best practice is to 
+use `timestamptz` instead. For why the use of `timestamp` is discouraged, see [PostgreSQL timestamp][postgresql-timestamp].
```
Comment (mkindahl) [inline use-timescale/hypertables/create.md]: It is possible to use integers as well for partitioning, even though they do not represent a time. This is not normally what we recommend though.

```suggestion
Hypertables are designed for real-time analytics and typically partitioned by columns that hold time values. These can be of the `timestamptz`, `date`, or `integer` types. While `timestamp` is also supported, a best practice is to 
use `timestamptz` instead. For why the use of `timestamp` is discouraged, see [PostgreSQL timestamp][postgresql-timestamp].
```

**Review by mkindahl — APPROVED** [review]

File: api/dimension_info.md [hunk 2]
```diff
@@ -65,6 +65,10 @@ The `partition_interval` should be specified as follows:
   `DATE`, this length should be specified either as an `INTERVAL` type
   or an integer value in *microseconds*.
 
+  <Highlight type="note">
+  While both `timestamp` and `timestamptz` data types are supported for partitioning columns, a best practice is to use `timestamptz`. For why the use of `timestamp` is discouraged, see [PostgreSQL timestamp](https://wiki.postgresql.org/wiki/Don't_Do_This#Don.27t_use_timestamp_.28without_time_zone.29).
```
Comment (billy-the-fish) [inline api/dimension_info.md]:
```suggestion
  While both `timestamp` and `timestamptz` data types are supported for partitioning columns, best practice is to use `timestamptz`. [PostgreSQL timestamp(https://wiki.postgresql.org/wiki/Don't_Do_This#Don.27t_use_timestamp_.28without_time_zone.29) explains why using `timestamp` is discouraged.
```

File: use-timescale/hypertables/create.md [hunk 3]
```diff
@@ -24,10 +24,8 @@ available](/api/:currentVersion:/hypertable/create_hypertable_old/).
 To create a hypertable, you need to create a standard PostgreSQL table, and then
 convert it into a hypertable.
 
-Hypertables are intended for time-series data, so your table needs a column that
-holds time values. This can be a timestamptz, date, or integer. Ensure that you
-set the datatype for the `time` column as `timestamptz` and not `timestamp`. For
-more information, see [PostgreSQL timestamp][postgresql-timestamp].
+Hypertables are designed for real-time analytics and partitioned by columns that hold time values. These can be of the `timestamptz`, `date`, or `integer` types. While `timestamp` is also supported, a best practice is to
```
Comment (billy-the-fish) [inline use-timescale/hypertables/create.md]:
```suggestion
Hypertables are designed for real-time analytics and partitioned by columns that hold time values. These can be of the `timestamptz`, `date`, or `integer` types. While `timestamp` is also supported, best practice is to 
```

File: use-timescale/hypertables/create.md (hunk 1 above)
Comment (billy-the-fish) [inline use-timescale/hypertables/create.md]:
```suggestion
use `timestamptz` instead. [PostgreSQL timestamp(https://wiki.postgresql.org/wiki/Don't_Do_This#Don.27t_use_timestamp_.28without_time_zone.29) explains why using `timestamp` is discouraged.
```

**Review by billy-the-fish — CHANGES_REQUESTED** [review]

**Review by billy-the-fish — APPROVED** [review]
LGTM

**Pull request merged**